staynomad/Nomad-Front
# Host Toggle

# Created host toggle

Closes #478 

# Summary of change

- Added toggle so that users can change from host to nonhost and vice versa under settings tab.


# Testing/Verification

- Go to settings under MyAccount.
- Press switch to host toggle.
- Make sure isHost is updated for the user in the mongodb collection.
- Non hosts should only have three tabs in MyAccount, while hosts should have five tabs.
